mongodb/templates/mongodb.conf.j2

```diff
@@ -0,0 +1,22 @@
+{% from "mongodb/map.jinja" import mongodb with context %}
+
+{# Bind to all ips(default) if in clustered mode,
+   otherwise only to the specified local ip.
+#}
+
+dbpath = /var/lib/mongodb
+logpath = /var/log/mongodb/mongodb.log
```

Comment:
These should be configurable. There is a high likelihood that other users of this formula might have their data volumes mounted at a different path.